portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase

```diff
@@ -0,0 +1,402 @@
+
+			CookieHelper._checkSelectedCookieValue(
+				cookieName = "GUEST_LANGUAGE_ID",
+				expectedCookieValue = "en_US");
+		}
+	}
+
+	@description = "This is a use case for LPS-170904 TC-4. Verify only functional cookies are set in Site level enable then disable, when the user isn't a first-time visitor."
+	@priority = 5
+	test VerifyFunctionalCookiesForNotFirstVisitInSiteLevelDisableImplicitMode {
+		property test.name.skip.portal.instance = "CookiesSiteLevelUseCase#VerifyFunctionalCookiesForNotFirstVisitInSiteLevelDisableImplicitMode";
+
+		task ("Given: Site Administrator enabled agreed on the cookie usage,then disable the cookie preference handling in site level.") {
+			for (var count : list "1,2") {
+				JSONUser.addUser(
+					userEmailAddress = "userea${count}@liferay.com",
+					userFirstName = "userfn${count}",
+					userLastName = "userln${count}",
+					userScreenName = "usersn${count}");
+
+				JSONUser.setFirstPassword(
+					agreeToTermsAndAnswerReminderQuery = "true",
+					requireReset = "false",
+					userEmailAddress = "userea${count}@liferay.com");
+
+				JSONUser.addUserToSite(
+					groupName = "Guest",
+					userEmailAddress = "userea${count}@liferay.com");
+			}
+
+			JSONRole.assignSiteRoleToUser(
+				groupName = "Guest",
+				roleTitle = "Site Administrator",
+				userEmailAddress = "userea1@liferay.com");
+
+			Site.openSiteSettingsAdmin(siteURLKey = "guest");
+
+			Site.addVirtualHostsURLCP(
+				pageVirtualHosts = "true",
+				pageVirtualHostURL = "www.able.com");
+
+			UserBar.signOut();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "userea1@liferay.com");
+
+			CookiePreferenceHandlingAdmin.enablePreferenceHandlingWithCookieConsentInSiteSettings(
+				baseURL = "http://www.able.com:8080",
+				portlet = "Preference Handling",
+				siteURLKey = "guest");
+
+			UserBar.signOut();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "userea2@liferay.com");
+
+			CookieBanner.cookiesBannerAcceptAll();
+
+			CookieHelper.checkAllCookieAreAcceptedAndSaved();
+
+			UserBar.signOut();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "userea1@liferay.com");
+
+			CookiePreferenceHandlingAdmin.disablePreferenceHandlingWithOutCookieConsentInSiteSettings(
+				baseURL = "http://www.able.com:8080",
+				portlet = "Preference Handling",
+				siteURLKey = "guest");
+		}
+
+		task ("When: End User second visits the site") {
+			UserBar.signOut();
+
+			Refresh();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "userea2@liferay.com");
+		}
+
+		task ("Then: Verify only the functional cookie is set cookie banner is present.") {
+			CookieBanner.verifyCookieBannerIsNotPresent();
+
+			CookieConsentPanel.verifyCookiesAreNotPresent();
```

Comment:
We might had to rename this to
- verifyCookiesForCookieCategoriesAreNotPresent();

Reply:
done